rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
@nrb Would it be possible to slim this override down to only the essential parts?